src/ORM/Table.php

```diff
@@ -2666,22 +2666,29 @@ public function callFinder(string $type, SelectQuery $query, mixed ...$args): Se
      */
     public function invokeFinder(Closure $callable, SelectQuery $query, array $args): SelectQuery
     {
+        // TODO this is where the missing shims need to be.
         $reflected = new ReflectionFunction($callable);
         $params = $reflected->getParameters();
         $secondParam = $params[1] ?? null;
-        $secondParamType = null;
+
+        $secondParamType = $secondParam?->getType();
+        $secondParamTypeName = $secondParamType instanceof ReflectionNamedType ? $secondParamType->getName() : null;
+
+        $secondParamIsOptions = (
+            count($params) === 2 &&
+            $secondParam?->name === 'options' &&
```

Comment:
Are there other variable names we should consider? The docs, app and bake all use `options` but there are likely other commonly used names.

Reply:
Supporting only the "official" variable name should suffice IMO.

Reply:
I think only `options` is good. It's fairly standardized.